lib/pages/metronome/metronome.dart

```diff
@@ -374,9 +413,17 @@ class _MetronomeState extends State<Metronome> {
   }
 
   Future<void> _startMetronome() async {
+    if (_deviceVolumeLevel == VolumeLevel.muted && _sound) {
```

Comment:
Use one if statement instead of two:
```suggestion
    if (_sound && [VolumeLevel.muted, VolumeLevel.low].contains(_deviceVolumeLevel)) {
```

Reply:
Done.